pallets/loans/benchmarking/src/lib.rs

```diff
@@ -222,34 +239,36 @@ benchmarks! {
     }
 
     liquidate_borrow {
-        let borrower: T::AccountId = whitelisted_caller();
-        initial_set_up::<T>(borrower.clone());
-        let caller: T::AccountId = account("Sample", 100, SEED);
-        <T as LoansConfig>::Currency::deposit(DOT, &caller.clone(), INITIAL_AMOUNT).unwrap();
-        let repay_amount = 2000;
-        let borrowed_amount = 100_000_000;
-        let currency_id: <T as ORMLOracleConfig<Instance1>>::OracleKey = T::convert(DOT);
+        let alice: T::AccountId = whitelisted_caller();
+        initial_set_up::<T>();
+        let bob: T::AccountId = account("Sample", 100, SEED);
+        let currency_id: <T as ORMLOracleConfig<Instance1>>::OracleKey = T::convert(KSM);
         let price: <T as ORMLOracleConfig<Instance1>>::OracleValue = T::convert_price(PriceWithDecimal{ price: FixedU128::from(100_000), decimal: 12 });
         assert_ok!(ORMLOracle::<T, _>::feed_values(SystemOrigin::Root.into(),
             vec![(currency_id, price)]));
-        assert_ok!(Loans::<T>::mint(SystemOrigin::Signed(borrower.clone()).into(), DOT, INITIAL_AMOUNT));
-        assert_ok!(Loans::<T>::collateral_asset(SystemOrigin::Signed(borrower.clone()).into(), DOT, true));
-        assert_ok!(Loans::<T>::borrow(SystemOrigin::Signed(borrower.clone()).into(), DOT, borrowed_amount));
-        let total_borrows = pallet_loans::TotalBorrows::<T>::get(DOT);
+        <T as LoansConfig>::Currency::deposit(DOT, &alice.clone(), 100_000_000_0).unwrap();
+        <T as LoansConfig>::Currency::deposit(KSM, &alice.clone(), 100_000_000_0).unwrap();
+        <T as LoansConfig>::Currency::deposit(DOT, &bob.clone(), 100_000_000_0).unwrap();
+        <T as LoansConfig>::Currency::deposit(KSM, &bob.clone(), 100_000_000_0).unwrap();
+        assert_ok!(Loans::<T>::mint(SystemOrigin::Signed(bob.clone()).into(), KSM, 200_000_000));
+        assert_ok!(Loans::<T>::mint(SystemOrigin::Signed(alice.clone()).into(), DOT, 200_000_000));
+        assert_ok!(Loans::<T>::collateral_asset(SystemOrigin::Signed(alice.clone()).into(), DOT, true));
+        assert_ok!(Loans::<T>::borrow(SystemOrigin::Signed(alice.clone()).into(), KSM, 100_000_000));
     }: {
-         let _ = Loans::<T>::liquidate_borrow(SystemOrigin::Signed(caller.clone()).into(), borrower, DOT, repay_amount, DOT);
+         let _ = Loans::<T>::liquidate_borrow(SystemOrigin::Signed(bob).into(), alice.clone(), KSM, 50_000_000, DOT);
```

Comment:
I don't think the `liquidate_borrow` can process successfully because Alice's shortfall is zero. You can let Alice have a positive shortfall by changing the price. https://github.com/parallel-finance/parallel/blob/master/pallets/loans/src/tests/liquidate_borrow.rs#L79